NetProbe/Model/NetworkMeasurement.mm

```diff
@@ -14,62 +15,108 @@
 static void setup_idempotent() {
     static bool initialized = false;
     if (!initialized) {
-
         // Set the logger verbose and make sure it logs on the "logcat"
-        mk::set_verbose(1);
-        // XXX Ok to call NSLog() from another thread?
```

Comment:
For the records: `NSLog()` is thread safe. Source: https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/Multithreading/ThreadSafetySummary/ThreadSafetySummary.html